src/cloud-cli/meltano/cloud/api/types.py

```diff
@@ -19,6 +19,7 @@ class CloudExecution(TypedDict):
     status: str
     exit_code: int
 
+    deployment_name: str
     environment_name: str
```

Comment:
Should the `environment_name` field be left behind?

Reply:
It's unused by the `history` cli so probably not. I went ahead and removed it.